I0604 14:33:35.960944 138143688118272 train.py:290] Training time window 1
Traceback (most recent call last):
  File "/home/pedro/PINN/jaxpi/examples/geomI_2ph/main.py", line 39, in <module>
    app.run(main)
  File "/usr/local/lib/python3.11/dist-packages/absl/app.py", line 308, in run
    _run_main(main, args)
  File "/usr/local/lib/python3.11/dist-packages/absl/app.py", line 254, in _run_main
    sys.exit(main(argv))
             ^^^^^^^^^^
  File "/home/pedro/PINN/jaxpi/examples/geomI_2ph/main.py", line 31, in main
    train.train_and_evaluate(FLAGS.config, FLAGS.workdir)
  File "/home/pedro/PINN/jaxpi/examples/geomI_2ph/train.py", line 369, in train_and_evaluate
    model = train_one_window(config, workdir, model, samplers, idx)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pedro/PINN/jaxpi/examples/geomI_2ph/train.py", line 129, in train_one_window
    model.state = model.step(model.state, batch)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/jaxpi/models.py", line 171, in step
    grads = grad(self.loss)(state.params, state.weights, batch, *args)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/jaxpi/models.py", line 124, in loss
    losses = self.losses(params, batch, *args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pedro/PINN/jaxpi/examples/geomI_2ph/models.py", line 366, in losses
    p_in_loss = jnp.mean((p_in_pred - self.p_in) ** 2)
                          ~~~~~~~~~~^~~~~~~~~~~
  File "/usr/local/lib/python3.11/dist-packages/jax/_src/numpy/array_methods.py", line 736, in op
    return getattr(self.aval, f"_{name}")(self, *args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/jax/_src/numpy/array_methods.py", line 264, in deferring_binary_op
    return binary_op(*args)
           ^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/jax/_src/numpy/ufuncs.py", line 85, in <lambda>
    fn = lambda x1, x2, /: lax_fn(*promote_args(numpy_fn.__name__, x1, x2))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: sub got incompatible shapes for broadcasting: (2048,), (141,).
--------------------
For simplicity, JAX has removed its internal frames from the traceback of the following exception. Set JAX_TRACEBACK_FILTERING=off to include these.
Waiting for JIT...